modules/patients/patientRouter.js

```diff
@@ -15,16 +15,21 @@ const {
   createPatient,
   editPatient,
   deletePatient,
+  patientByDoctor,
 } = require('./patientService');
 
 router.get('/', authentication, patientList);
+
+router.get('/doctor/:userId', patientByDoctor);
```

Comment:
authentication ?  and whose userId is this ?

Reply:
modified